banzai_nres/settings.py

```diff
@@ -16,9 +16,10 @@
                   'banzai.uncertainty.PoissonInitializer',
                   'banzai.dark.DarkSubtractor',
                   'banzai_nres.flats.FlatLoader',
-                  'banzai_nres.background.BackgroundSubtractor',
+                  # 'banzai_nres.background.BackgroundSubtractor',
```

Comment:
Hmm. So we are not doing background subtraction? I guess we can leave this, but we need to test if actually makes things worse.

Reply:
removed.